src/metrics.py

```diff
@@ -73,7 +73,26 @@ class RunnerInstalled(Event):
     """
 
     flavor: str
-    duration: NonNegativeInt
+    duration: NonNegativeFloat
+
+
+class RunnerStart(Event):
+    """Metric event for when a runner is started.
+
+    Attrs:
+        flavor: Describes the characteristics of the runner.
+          The flavour could be for example "small".
```

Comment:
very nitpicky, no need to resolve - you have flavor and flavour, so mixing the two spellings ;)